blocked-edges/4.18.0-ec.3-CMOSuddenStrictConfigValidation.yaml

```diff
@@ -0,0 +1,16 @@
+to: 4.18.0-ec.3
```

Comment:
well, I think this is not enough, an upgrade from `4.18.0-ec.0` (where 4.17.5 wasn't reached/available) to `4.18.0-ec.4` will also suffer from this.
I'll need to think about this.

Reply:
That's not too hard - you'll just need a `4.18.0-ec.4` file with `from: 4.18.0-ec.0`,

Reply:
I only gave an example of an edge I was missing, IIUC we actually need to cover

(Given/assuming the `from` skipped `4.17.5`)

```
4.17 ECs and RCs + 4.17.[0-4] + 4.18.0-ec.[0-2] -> 4.18.0-ec.3
4.18.0-ec.[0-2] -> 4.18.0-ec.4
4.18.0-ec.[0-2] -> 4.18.0-rc.Z (every Z)
4.18.0-ec.[0-2] -> 4.18.Z' (every Z')
4.18.0-ec.[0-2] -> 4.19.Z" (every Z")
```

I'll add files for the available ones and I think you have automation for the future versions (that being said, I can get a head and add some for the next ones until the automation is set up)

Reply:
```
4.17 ECs and RCs + 4.17.[0-4] + 4.18.0-ec.[0-2] -> 4.18.0-ec.3
4.18.0-ec.[0-2] -> 4.18.0-ec.4
4.18.0-ec.[0-2] -> 4.18.0-rc.Z (every Z)
```
These are fine

```
4.18.0-ec.[0-2] -> 4.18.Z' (every Z')
```
We will only need the one for 4.18.0 - we force pre-release version to go through the GA one.

```
4.18.0-ec.[0-2] -> 4.19.Z" (every Z" + ECs and RCs )
```
This will not be necessary, 4.19.0-ec.0 will only have [an edge from 4.18.0-rc.0](https://github.com/openshift/cincinnati-graph-data/blob/master/build-suggestions/4.19.yaml#L2C3-L2C25)

Reply:
Great, thanks for the details.